src/io/obf.rs

```diff
@@ -63,34 +62,11 @@ where
         obf_params.encoding_sigma,
     );
     let s_init = &bgg_encode_sampler.secret_vec;
-    let t_bar_matrix = sampler_uniform.sample_uniform(&params, 1, 1, DistType::FinRingDist);
-    log_mem("Sampled t_bar_matrix");
-
-    let hardcoded_key_matrix = sampler_uniform.sample_uniform(&params, 1, 1, DistType::BitDist);
-    log_mem("Sampled hardcoded_key_matrix");
-
-    let enc_hardcoded_key = {
-        let e = sampler_uniform.sample_uniform(
-            &params,
-            1,
-            1,
-            DistType::GaussDist { sigma: obf_params.hardcoded_key_sigma },
-        );
-        let scale = M::P::from_const(&params, &<M::P as Poly>::Elem::half_q(&params.modulus()));
-        t_bar_matrix.clone() * &public_data.a_rlwe_bar + &e -
-            &(hardcoded_key_matrix.clone() * &scale)
-    };
-    let enc_hardcoded_key_polys = enc_hardcoded_key.get_column_matrix_decompose(0).get_column(0);
-    log_mem("Sampled enc_hardcoded_key_polys");
-
-    let t_bar = t_bar_matrix.entry(0, 0);
-    #[cfg(feature = "test")]
-    let hardcoded_key = hardcoded_key_matrix.entry(0, 0);
 
     let mut plaintexts = (0..obf_params.input_size.div_ceil(dim))
         .map(|_| M::P::const_zero(params.as_ref()))
         .collect_vec();
-    plaintexts.push(t_bar.clone());
+    plaintexts.push(t.entry(0, 0).clone());
```

Comment:
If the `t` here only contains one random polynomial, this is called `t_bar` in the paper.
In the contrast, a vector `t` is defined by `(t_bar, -1)`.